React/Base/RCTConvert.m

```diff
@@ -735,14 +739,16 @@ + (UIImage *)UIImage:(id)json
                           orientation:image.imageOrientation];
   }
 
+#if !TARGET_OS_TV
```

Comment:
That's a good question -- it seems to compile for tvOS now that I'm on tvOS 10.  Will back this out.